app/services/follow_remote_account_service.rb

```diff
@@ -16,7 +16,7 @@ def call(uri)
     return Account.find_local(username) if TagManager.instance.local_domain?(domain)
 
     account = Account.find_remote(username, domain)
-    return account unless account.nil?
+    return account unless account&.last_webfingered_at.nil? || account.last_webfingered_at + 86_400 < Time.now.utc
```

Comment:
As for 1.day.from_now, that indeed seems much more readable, but I don't know RoR enough to understand how it handles timezones